packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -72,8 +70,11 @@ const PersonalDetailsForm = props => {
         };
     };
 
+    const is_for_idv_or_onfido = is_for_idv || is_for_onfido;
+    const is_not_for_idv_and_onfido = !is_for_idv && !is_for_onfido;
```

Comment:
cant we use `const is_not_for_idv_and_onfido = !is_for_idv_or_onfido`

Reply:
@amina-deriv we need both `is_for_idv` and `is_for_onfido` to be false